staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -236,23 +236,23 @@ func (p *Patcher) Patch(current runtime.Object, modified []byte, source, namespa
 		}
 		current, getErr = p.Helper.Get(namespace, name)
 		if getErr != nil {
-			return nil, nil, getErr
+			return nil, getErr
 		}
-		patchBytes, patchObject, err = p.patchSimple(current, modified, namespace, name, errOut)
+		patchObject, err = p.patchSimple(current, modified, namespace, name, errOut)
 	}
 	if err != nil {
 		if (apierrors.IsConflict(err) || apierrors.IsInvalid(err)) && p.Force {
-			patchBytes, patchObject, err = p.deleteAndCreate(current, modified, namespace, name)
+			patchObject, err = p.deleteAndCreate(current, modified, namespace, name)
 		} else {
 			err = cmdutil.AddSourceToErr("patching", source, err)
 		}
 	}
-	return patchBytes, patchObject, err
+	return patchObject, err
 }
 
-func (p *Patcher) deleteAndCreate(original runtime.Object, modified []byte, namespace, name string) ([]byte, runtime.Object, error) {
```

Comment:
It is curious that in the original code the `modified` parameter is always returned unmodified. I think it is good that you've removed it from this function.